apps/storefront/components/Navbar/Navbar.tsx

```diff
@@ -24,10 +25,12 @@ export function Navbar() {
   const { authenticated } = useAuthState();
   const { checkout } = useCheckout();
   const { currentLocale } = useRegions();
+  const { checkoutUrl } = useSaleorAppSettings();
 
-  const externalCheckoutUrl = checkout
-    ? `/checkout/?checkout=${checkout.id}&locale=${currentLocale}`
-    : "#";
+  const externalCheckoutUrl =
+    checkout && checkoutUrl
+      ? `${checkoutUrl}/?checkout=${checkout.id}&locale=${currentLocale}`
+      : "#";
```

Comment:
is this the only place where checkout redirects? Should this logic be extracted somewhere?